src/main/java/at/hannibal2/skyhanni/features/itemabilities/CrownOfAvariceCounter.kt

```diff
@@ -39,28 +45,44 @@ object CrownOfAvariceCounter {
     private var display: List<Renderable> = emptyList()
     private val MAX_AVARICE_COINS = 1.billion
     private val MAX_AFK_TIME = 2.minutes
+    private var inventoryOpen = false
     private val isWearingCrown by RecalculatingValue(1.seconds) {
         InventoryUtils.getHelmet()?.getInternalNameOrNull() == internalName
     }
 
     private var count: Long? = null
     private var coinsEarned: Long = 0L
-    private var sessionStart: SimpleTimeMark? = null
+    private var sessionStart: Duration = 0.seconds
```

Comment:
does this still represent the start time? idk what a start duration would be. mabe rename to session uptime or someting if this is now just the duration of the session